easybuild/easyconfigs/o/openCARP/openCARP-17.0-foss-2023b.eb

```
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'openCARP'
+version = '17.0'
+
+homepage = 'https://opencarp.org'
+description = "openCARP is an open cardiac electrophysiology simulator for in-silico experiments."
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = ['https://git.opencarp.org/openCARP/openCARP/-/archive/v%(version)s/']
+sources = ['openCARP-v%(version)s.tar.gz']
+checksums = [
+    '6439b6b8e6a265cd6cfe39f59507418739d936de6f6e059b2d0f468df816d26e',  # openCARP-v17.0.tar.gz
+]
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('gengetopt', '2.23'),
+    ('pkgconf', '2.0.3'),
+]
+
+dependencies = [
+    ('PETSc', '3.22.3'),
+    ('zlib', '1.2.13'),
+]
+
+
+configopts = "-DSPACK_BUILD=ON "
```

```suggestion
configopts = "-DSPACK_BUILD=ON "
```
I looked into what this implies and found https://git.opencarp.org/openCARP/openCARP/-/blob/master/cmake/postinst?ref_type=heads , there are hard-coded directories in there that are not portable and almost certainly do not exist...so I think this step will only print the final warning in that file.

---

It would also avoid include this file used for CPack packaging: https://git.opencarp.org/openCARP/openCARP/-/blob/master/cmake/CPackConfig.cmake?ref_type=heads

---

We are working on refactoring this installation part that is not portable in openCARP, but we use this workaround for now to be able to use the install target.

---

@MarieHouillon What's the impact of enabling this `SPACK_BUILD` option?

---

It disables some postinstall steps used for packaging

---

OK, then a line of comment to mention that makes sense, because this looks very out of place in a build recipe for EasyBuild :)

---

I'll add it. We will get rid of this flag anyway in a near future.